### PR TITLE
Updated requirements.txt with `pygame-ce`

### PR DESCRIPTION
**Pull request by Matiiss**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pygame>=2.1.2
+pygame-ce>=2.1.3
 click>=8.1.3
 colorama>=0.4.4
-keyboard>=0.13.5
\ No newline at end of file
+keyboard>=0.13.5
```
Comment (Matiiss) [inline requirements.txt:4]: No clue what's going on with this last line, probably something about a newline getting removed or maybe added.

Comment (Grimmys) [reply requirements.txt:4]: You can enable an option to hide this kind of changes on Github preview:
image
Then you can focus on the meaningful changes.

Comment (Matiiss) [reply requirements.txt:4]: Thanks for the tip. 👍

**Review by pycoinfu — APPROVED** [review]
LGTM, thanks

**Pull request merged**